pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```diff
@@ -140,7 +157,16 @@ async def _run_for(s3_path):
             await exectools.cmd_assert_async(cmd)
 
         await _run_for(release_path)
-        await _run_for(latest_path)
+
+        # make sure that the latest path has the same microshift build as the given release assembly
+        # only then sync the pullspec to the latest path
+        latest_packages_path = f"{latest_path}/os/Packages/"
+        if await self.is_microshift_for_release_in_latest(latest_packages_path):
```

Comment:
Will the microshift_bootc job triggered by microshift job? if its triggered by microshift job can this be a parameter of the job to update latest?

Reply:
Yes the `build-microshift-bootc` job is triggered by the `build-microshift` job.
So you're proposing that build-microshift job determine if latest needs update or not? And then sets the flag
Or are you proposing that someone set that flag by hand?
The enhancement here is to remove the manual part, i.e. we can do this basic check and only 
update latest when rpms are sync'd first

Reply:
I kind of get the propose you block microshift-sync job above the wait it populates the path? 
yeah, I think set a `--latest` arg will be simpler